submission-history-database/schema.sql

```diff
@@ -0,0 +1,16 @@
+CREATE SCHEMA submissionHistory;
+USE submissionHistory;
+
+CREATE TABLE submissionTable(
+
+submission int PRIMARY KEY AUTO_INCREMENT, 
+userID INT(255), 
+userName VARCHAR(255), 
+challengeId INT(255), 
+challengeName VARCHAR(255), 
+programmingLanguage CHAR(255), 
```

Comment:
We can change this to VARCHAR to reduce the size of the database

Reply:
Resolved